app/assets/src/components/views/SampleViewV2/report_table.scss

```diff
@@ -57,28 +57,38 @@
   font-weight: $font-weight-semibold;
   text-align: left;
 
-  .taxonName {
-    text-transform: capitalize;
-  }
-
   .expandIcon {
     flex: 0 0 20px;
   }
 
-  .missingName {
-    color: $medium-grey;
-  }
-}
+  .taxonContainer {
+    .taxonName {
+      text-transform: capitalize;
+    }
 
-.genusRow {
-  background-color: $off-white;
+    .missingName {
+      color: $medium-grey;
+    }
+
+    .countInfo {
+      // word-wrap: break-word;
```

Comment:
Is this meant to be commented out?

Reply:
No. I was going to remove it. But i guess we can leave for long words.

Reply:
Not commented, but removed. `white-space: normal` takes care of the wrapping.
Break-word is for very long word which does not happen in this case.